packages/prisma/src/prisma/schema.prisma

```diff
@@ -120,22 +126,27 @@ model User {
   session           Session[]
   accounts          Account[]
   courses           Course[]
-  questions         Question[]
+  questions         Element[]
   mediaFiles        MediaFile[]
   tags              Tag[]
   questionInstances QuestionInstance[]
   sessions          LiveSession[]
   learningElements  LearningElement[]
   microSessions     MicroSession[]
   groupActivities   GroupActivity[]
+  elementInstances  ElementInstance[]
```

Comment:
Why do we need a link from the user to all ElementInstances?

Reply:
same as we have one from user to question instance, maybe because of bookmarks or responses and responsedetails?

Reply:
this is added automatically as we have owner/ownerId on the ElementInstance. just the other side of the relation

Reply:
For what do we need the owner on the ElementInstance then? We didn't need one on the QuestionInstance?